src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -421,15 +432,9 @@ export const FileUpload:React.FC<IFileUploadProps>=({
                     <input {...getInputProps()}  />
                 </SubContainer>
             </Container>
-            <BottomPanel withBorder padding='30px 20px 43px 20px' opacity={state.loading.opacity} position={state.loading.position} ref={loadingContainerRef} overflow='hidden' width={state.componentWidth} margin={`${state.margin}px`} positionTop={state.positionTopLoading}>
-                <Loading loading={isUploading} message='Uploading...' />
-            </BottomPanel>
-            <BottomPanel withBorder opacity={state.isSuccess.opacity} height={state.loadingContainerHeight} position={state.isSuccess.position} margin={`${state.margin}px`} positionTop={state.positionTopLoading} width={state.isSuccessWidth}>
-                <IsFailureIsSuccessPanel message={successMessage} iconColor={MainTheme.colors.statusColors.green} IconToShow={CheckCircle} />
+            <BottomPanel withBorder padding={isFailure||isSuccess?undefined:'30px 20px 43px 20px'} opacity={state.loading.opacity||state.isSuccess.opacity||state.isFailure.opacity} position={state.loading.position&&state.isFailure.position&&state.isSuccess.position} ref={loadingContainerRef} overflow='hidden' width={isFailure||isSuccess?state.isSuccessWidth:state.componentWidth} margin={`${state.margin}px`} positionTop={state.positionTopLoading} height={isFailure||isSuccess?state.loadingContainerHeight:undefined}>
+                {renderChild()}
```

Comment:
This no longer makes sense to have all these opacities here. As we talked about yesterday this Bottompanel is a container for the children rendered inside of it. Create this panel and transition the opacity and height, it is either displayed or not.

Now, opacity needs to be added to the rendered components. I think this can be achieved by adding a @keyframes to the IsFailureIsSuccessPanel for opacity from 0 to 1 with 0.5s animation. There may be another way to do this.

Padding should also be static, I don't see why it would need to resize.

Add appropriate CSS to BottomPanel/IsFailureIsSuccess so that elements are not at edge of div.